Add 1 on 1 video calling/client.js

```diff
@@ -0,0 +1,159 @@
+import { CallClient, CallAgent, Renderer, LocalVideoStream} from "@azure/communication-calling";
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+
+let call;
+let callAgent;
+const calleeInput = document.getElementById("callee-id-input");
+const callButton = document.getElementById("call-button");
+const hangUpButton = document.getElementById("hang-up-button");
+const stopVideoButton = document.getElementById("stopVideo");
+const startVideoButton = document.getElementById("startVideo");
+
+let placeCallOptions;
+let deviceManager;
+let localVideoStream;
+let remoteVideoStream;
+let rendererLocal;
+let rendererRemote;
+
+function handleVideoStreams(v) {
+  remoteVideoStream = v;
+  remoteVideoView();
+  remoteVideoStream.on('availabilityChanged', async () => {
+    if (remoteVideoStream.isAvailable) {
+        remoteVideoView();
+    } else {
+        rendererRemote.dispose();
+    }
+  });
+  if (remoteVideoStream.isAvailable) {
+    remoteVideoView();
+  }
+}
+
+function subscribeToRemoteParticipant(p) {
+  p.videoStreams.forEach(v => {
+    handleVideoStreams(v);
+  });
+  p.on('videoStreamsUpdated', e => {
+    e.added.forEach(v => {
+      handleVideoStreams(v);
+    })
+  });
+}
+
+
+async function init() {
+  const callClient = new CallClient();
+  const tokenCredential = new AzureCommunicationTokenCredential("<USER ACCESS TOKEN>");
+  callAgent = await callClient.createCallAgent(tokenCredential, { displayName: 'optional ACS user name' });
+      
+  deviceManager = await callClient.getDeviceManager();
+  callButton.disabled = false;
+
+  callAgent.on('incomingCall', async e => {
+    const videoDevices = await deviceManager.getCameras();
+    const videoDeviceInfo = videoDevices[0];
+    localVideoStream = new LocalVideoStream(videoDeviceInfo);
+    localVideoView();
```

Comment:
can this take param localVideoStream? so you don't have to keep it as global?

Reply:
localVideoStream will still need to be global since it is referenced when toggling buttons.